R/appMetadata.R

```diff
@@ -275,6 +274,14 @@ appIsDocument <- function(appMode) {
   )
 }
 
+appIsQuartoDocument <- function(appMode) {
```

Comment:
nit: Because this might be a single document or a more complicated project; maybe `appIsQuartoContent`?

Reply:
Words are hard, eh? Other thoughts: `isQuartoContent`, `appModeIsQuarto`, etc.

Reply:
I just modified from the `appIsDocument()`  above, so while I agree that the name isn't the best, I'll leave it as for this PR.